docs/netdata-agent/versions-and-platforms.md

```diff
@@ -67,3 +67,49 @@ Static builds usually miss certain features that require operating-system suppor
 - eBPF related features
 
 When platforms are removed from the [Binary Distribution Packages](/packaging/makeself/README.md) list, they default to install or update Netdata to a static build. This may mean that after platforms become EOL, Netdata on them may lose some of its features. We recommend upgrading the operating system before it becomes EOL, to continue using all the features of Netdata.
+
+### Upgrading on an Unsupported Platform
+
+Existing installs of native packages on a platform that is no longer supported, cannot be automatically updated at this time. The upgrader will show it is already at the newest version, because we no longer publish new packages.
+
+If the operating system cannot be upgraded to a more recent version, the install can be switched to a static build. This is a manual process, may cause data loss, and is therefore **not supported**. However, following these steps should result in a functioning Agent with metrics data and connection to Netdata Cloud in tact:
+
+1. Stop the Agent, as appropriate for your platform. For platforms with systemd:
+
+   ```sh
+   sudo systemctl stop netdata
+   ```
+
+2. Backup the contents of these directories: `/etc/netdata`, `/var/cache/netdata`, `/var/lib/netdata`, `/var/log/netdata`.
```

Comment:
```suggestion
2. Backup the contents of these directories: `/INSTALL_PREFIX/netdata`, `/var/cache/netdata`, `/var/lib/netdata`, `/var/log/netdata`.
```

Do native packages install _always_ on `/etc`? If yes, ignore this comment.

Reply:
Yes, they do.